### PR TITLE
feat: Add databricks notebooks

### PR DESCRIPTION
**Pull request by ftoft**
## References

* #32

### CODE REVIEW

File: source/databricks/tox.ini [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Configure flake8 to fit development environment
 [flake8]
-ignore = E402, F401, W503
+ignore = E402, F401
```
Comment (BjarkeMeier) [inline source/databricks/tox.ini:3]: Is it intended to remove W503?

File: source/notebooks/balance-fixing-poc.py [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+# - Use stored time in parquet rather than enqueued time from EventHub as enqueued time might differ from stored time and thus cause errors when e.g. fetching basis data
+
+# General (future?) considerations
+# - In geh-timeseries: Why not enrich time series with metering point data (or simply publish the metering point data that was used for validation of the metering points)?
+#   Then we would not have to deal with metering point events in wholesale and perhaps more important both domains will have the same perspective about the points
+# - There seems to be a problem with the modelling of grid areas vs grid area links in the actor register.
+#   How can a grid area have a (single) grid area link id when it's a one-to-many relation?
+
+# Job parameters
+from datetime import datetime
```
Comment (BjarkeMeier) [inline source/notebooks/balance-fixing-poc.py:29]: Is this the latest version of the notebook?

File: source/notebooks/balance-fixing-poc.py [hunk 3]
```diff
@@ -0,0 +1,248 @@
+# Databricks notebook source
```
Comment (BjarkeMeier) [inline source/notebooks/balance-fixing-poc.py:1]: How about adding a README.md in the notebooks folder explaining how you recommend setting up the necessary environment variables with secrets on the cluster?

Comment (ftoft) [reply source/notebooks/balance-fixing-poc.py]: I have made a README.md in the notebook folder, will you check it out?

File: .licenserc.json [hunk 4]
```diff
@@ -93,6 +93,21 @@
       "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.",
       "# See the License for the specific language governing permissions and",
       "# limitations under the License."
+    ],
+    "**/*.scala": [
+      "# Copyright 2020 Energinet DataHub A/S",
```
Comment (BjarkeMeier) [inline .licenserc.json:98]: Are these valid scala comments?

File: source/notebooks/balance-fixing-poc.py (hunk 3 above)
Comment (BjarkeMeier) [reply source/notebooks/balance-fixing-poc.py]: Looks good 👍

File: source/notebooks/issue-32/add-gridarea-805-806-to-eventhub.scala [hunk 5]
```diff
@@ -0,0 +1,100 @@
+// Databricks notebook source
+// Copyright 2020 Energinet DataHub A/S
```
Comment (BjarkeMeier) [inline source/notebooks/issue-32/add-gridarea-805-806-to-eventhub.scala:2]: I'm a little concerned that there is no license check complaints when you use different comment markers in the license check configuration and in the notebook file.

Comment (ftoft) [reply source/notebooks/issue-32/add-gridarea-805-806-to-eventhub.scala:2]: I have fixed it in configuration

File: .licenserc.json (hunk 4 above)
Comment (ftoft) [reply .licenserc.json]: I fixed it to match scala

File: source/notebooks/issue-32/add-gridarea-805-806-to-eventhub.scala (hunk 5 above)
Comment (ftoft) [reply source/notebooks/issue-32/add-gridarea-805-806-to-eventhub.scala:2]: Now it works, added license check to .github repo

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**